### PR TITLE
Fix typos for doc in erts

### PR DESCRIPTION
**Pull request by kianmeng**

### CODE REVIEW

File: erts/doc/src/notes.xml [hunk 1]
```diff
@@ -16200,7 +16200,7 @@
           <p>
 	    The effect of the deprecated environment variable
 	    ERL_MAX_PORTS had been removed premeturely. It has now
-	    been readded. Note that this is still scheduled to be
+	    been read. Note that this is still scheduled to be
```
Comment (garazdawi) [inline erts/doc/src/notes.xml:16203]:
```suggestion
	    been re-added. Note that this is still scheduled to be
```

**Review by garazdawi — CHANGES_REQUESTED** [review]
Thanks for this! I just found a small mistake that you did, when you fix that we can merge this.

File: erts/doc/src/notes.xml (hunk 1 above)
Comment (sverker) [reply erts/doc/src/notes.xml:16203]: While at it, fix misspelled "prem**a**turely" in the line above.

**Comment by garazdawi** [comment]
Thanks!

**Pull request merged**